src/components/popover-container/popover-container.component.js

```diff
@@ -53,20 +46,24 @@ const PopoverContainer = ({
             data-element='popover-container-content'
             animationState={ state }
             position={ position }
+            isStickyTop={ isStickyTop }
```

Comment:
shouldn't it be `hasStickyTop`?

Reply:
I thought in this way: Is this sticky top, not in the way: has this component sticky top?